README.md

```diff
@@ -1,24 +1,196 @@
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 SummerWave Hotels (Backend) <a id="about-project"></a>
 
-* Database creation
+**SummerWave Hotels** is a website to reserve rooms for the SummerWave Hotel chains. You can select available rooms from a gaphic list in the home page of the website, or you can just click on the reserve menu item and see the full list of available rooms and the city in which they are available. You can find the repository for the [front end ](https://github.com/dev89s/final-capstone-front-end)here.
 
-* Database initialization
+Backend is implemented as API-only. The front end is implemented utilizing the React framework.
 
-* How to run the test suite
+You can find the setup of the Kanban board [here](https://github.com/abeltsew/final_capstone_be/issues/13).
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🛠 Built With <a id="built-with"></a>
 
-* Deployment instructions
+### Tech Stack <a id="tech-stack"></a>
 
-* ...
+<details>
+  <summary>Ruby on Rails</summary>
+  <ul>
+    <li><a>https://rubyonrails.org/</a></li>
+    <!-- <li><a>https://rubyonrails.org/</a></li> -->
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a id="key-features"></a>
+
+- **Ability to see the featured rooms in the homepage**
+- **The ability to reserve rooms based by selecting the city, date and room name**
+- **The ability to add rooms the the database**
+- **The ability to deactivate rooms and make them unavailable to reserve**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a id="live-demo"></a>
+
+You can see a live demo for the backend-api [here](https://summerwave.onrender.com/).
+You can find the documentation for the backend api [here](https://summerwave.onrender.com/api-docs/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ## 🎬 Representaition <a id="representation"></a> -->
+
+<!-- not added -->
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a id="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install Ruby
+- Install Rails
+<!-- - Install Rspec -->
+- Install Postgresql
+<!-- - Install React -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git clone https://github.com/abeltsew/final_capstone_be.git
+```
+
+### Install
+
+Install this project with:
+
+```
+  cd final_capstone_be
+  bundle install
+```
```

Comment:
- [ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: ``rails db:create, rails db:migrate, rails db:seed``. 👍